lib/opencensus/stats/exporters/stackdriver.rb

```diff
@@ -135,7 +139,8 @@ def initialize \
           else
             credentials = final_credentials credentials, scope
             @client_promise = create_client_promise \
-              @executor, credentials, scope, client_config, timeout
+              @executor, credentials, scope, client_config, timeout,
```

Comment:
Note: this code path was not covered by the rake tests, because they set `mock_client`.
This code depends on the newer version of `google-cloud-monitoring` (at least 0.30.0), while the current gemspec pulls in 0.29.2.
I've sent #26 to update the dependency.